X/xxHash_jll/build_tarballs.jl

```diff
@@ -0,0 +1,51 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "xxHash_jll"
```

Comment:
Oh, I see the problem: you appended `_jll` here, that's why the wizard created a new file